src/Api/Struct/Webspace/PhpSettings.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Copyright 1999-2019. Parallels IP Holdings GmbH.
+
+namespace PleskX\Api\Struct\Webspace;
+
+class PhpSettings extends \PleskX\Api\Struct
+{
+    /** @var array */
+    public $properties;
+
+    public function __construct($apiResponse)
+    {
+        $this->properties = [];
```

Comment:
Use $this->_initScalarProperties here, see \PleskX\Api\Struct\Webspace\DiskUsage::__construct

Reply:
Even in Plesk we don't know about all php settings available - it would be useless if we enumerate only a couple of them

Reply:
Good, it sounds reasonable. In any case, it is better to change the method signature to make it look like the others.